manifests/foobar2000/foobar2000/1.5.4.Yaml

```diff
@@ -0,0 +1,18 @@
+Id: foobar2000.foobar2000
+Version: 1.5.4
+Name: foobar2000
+Publisher: foobar2000
+License: foobar2000 license 
+LicenseUrl: https://www.foobar2000.org/license
+AppMoniker: foobar2000
+Tags: music
+Description: foobar2000 music player
+Homepage: https://www.foobar2000.org/
+InstallerType: exe
+Installers:
+  - Arch: x86
+    Url: http://www.foobar2000.org/files/23ded734ebe6fd79f0ce3e50ee03cc84/foobar2000_v1.5.4.exe
+    Sha256: 838ADC84C9D36312F82E333C8876CE9B653436F33912A00F879E52F99A1C8654
+    Switches:
+      Silent: /S
+      SilentWithProgress: /S
```

Comment:
```suggestion
InstallerType: nullsoft
Installers:
  - Arch: x86
    Url: http://www.foobar2000.org/files/23ded734ebe6fd79f0ce3e50ee03cc84/foobar2000_v1.5.4.exe
    Sha256: 838ADC84C9D36312F82E333C8876CE9B653436F33912A00F879E52F99A1C8654
```

Reply:
This is a nullsoft installer, so the nullsoft type can be used. No flags are necessary for the nullsoft type.